wallets/errors.js

```diff
@@ -0,0 +1,22 @@
+export class InvoiceCanceledError extends Error {
```

Comment:
These errors in `components/payment.js` were causing a cyclical dep so I yanked them out.